src/quick-lint-js/parse.h

```diff
@@ -1482,6 +1484,10 @@ class parser {
           //   method() {}
           // }
           v.visit_property_declaration(property_name);
+        } else if (async_static) {
+          this->error_reporter_->report(error_async_static_method{
```

Comment:
Document with a comment:
```suggestion
        } else if (async_static) {
          // async static method() {}             // Invalid.
          // async static *myAsyncGenerator() {}  // Invalid.
          this->error_reporter_->report(error_async_static_method{
```